packages/cli/internal/pkg/aws/cdk/progress.go

```diff
@@ -51,13 +51,13 @@ func (p ProgressStream) DisplayProgress(description string) error {
 	return nil
 }
 
-func ShowExecution(progressStreams []ProgressStream) []Result {
+func PerformExecution(progressStreams []ProgressStream, showLogging bool) []Result {
```

Comment:
I'm approving this as is.

If you want to use the if statement with the bool arg and make this even cleaner by consolidating the 3 functions, you may consider making bool arg optional with default. See Option 2 in this post: https://joneisen.me/programming/2013/06/23/golang-and-default-values.html

Reply:
This function should be private now since it should be accessed by the two new functions below.